packages/expo-modules-core/ios/NativeModulesProxy/EXNativeModulesProxy.mm

```diff
@@ -74,6 +74,18 @@ - (instancetype)initWithModuleRegistry:(nullable EXModuleRegistry *)moduleRegist
   return self;
 }
 
+/**
+ The designated initializer is for Expo Go to pass a custom `EXModuleRegistry`
```

Comment:
If it calls another init using `self` instead of `super`, it's the convenience initializer, not designated. I would rather just skip this word.

```suggestion
 The initializer for Expo Go to pass a custom `EXModuleRegistry`
```

Reply:
good to know. i never know the name _convenience initializer_ before 😅 thanks!

Reply:
You should write more Swift 😄 It's the keyword there, whereas in Objective-C it's not, so devs usually don't know that such distinction exists.